Makefile

```
@@ -23,11 +23,11 @@ clean:
 
 ## coverage - Generate coverage reports for the project
 coverage:
-	./generate_test_reports.sh
+	bash scripts/unix/generate_test_reports.sh
```

We should just be able to call these directly?

```suggestion
	./scripts/unix/generate_test_reports.sh
```

---

I added them 
1) to make sure they run with the correct shell (yes, I know the shebang handles this, but see 2)
2) to make sure they run in case the scripts have not been giving the execute permission (`chmod +x`).